src/index.html

```diff
@@ -142,7 +158,52 @@ <h3 class="modal__title">New Post</h3>
         <p class="modal__caption"></p>
       </div>
     </div>
-    <script src="./scripts/validation.js"></script>
-    <script src="./scripts/index.js"></script>
+
+    <div class="modal" id="avatar-modal">
+      <div class="modal__container">
+        <button type="button" class="avatar__close-btn"></button>
```

Comment:
`avatar__close-btn`

Every **close icon** should have a **common** `modal__close-btn` class.


The idea, why all common elements should have **common** css classes, is that you can create common `css` styles, common **universal** functions, find all common elements in the DOM and so on…

Also there should be common classes for `close` button, `form`, `input`, `submit` button.